lib/player.js

```diff
@@ -133,6 +133,9 @@ shaka.Player = function(video, dependencyInjector) {
   /** @private {boolean} */
   this.deferredVariantClearBuffer_ = false;
 
+  /** @private {number} */
+  this.deferredVariantClearBufferOffset_ = 0;
```

Comment:
"Deferred" is fine here, but again, I think it should include safeMargin instead of offset.